pkgs/python-packages/pyln-proto/default.nix

```
@@ -27,6 +29,7 @@ buildPythonPackage rec {
   postUnpack = "sourceRoot=$sourceRoot/contrib/pyln-proto";
   postPatch = ''
     sed -i '
+      s|pycparser==2.20|pycparser>=2.20|
```

```suggestion
      s|pycparser==2.20|pycparser~=2.20|
```
This is safer because it rejects potentially incompatible major version updates.